engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/AvroSchemaDeterminer.scala

```diff
@@ -34,3 +35,8 @@ object RuntimeSchemaData {
 }
 
 class SchemaDeterminerError(message: String, cause: Throwable) extends RuntimeException(message, cause)
+
+case object FixedNoneSchemaDeterminer extends AvroSchemaDeterminer {
```

Comment:
I think that we don't need `FixedNoneSchemaDeterminer` but `FixedStringSchemaDeterminer`, because at the end this schema will be used to determine type of key. Also I would add here a description that this determiner is because we want to have one generic source that will handle both situations: key with defined schema in registry and key as fixed string type.

Reply:
Changed to FixedStringSchemaDeterminer (have not removed, it's used in Specific source)